src/EasyAbp.BookingService.Domain/EasyAbp/BookingService/Assets/Asset.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using EasyAbp.BookingService.AssetCategories;
+using EasyAbp.BookingService.AssetSchedules;
+using JetBrains.Annotations;
+using Volo.Abp.Domain.Entities.Auditing;
+using Volo.Abp.MultiTenancy;
+
+namespace EasyAbp.BookingService.Assets;
+
+public class Asset : FullAuditedAggregateRoot<Guid>, IMultiTenant
+{
+    public virtual Guid? TenantId { get; protected set; }
+    
+    [NotNull]
+    public virtual string Name { get; protected set; }
+    
+    [NotNull]
+    public virtual string AssetDefinitionName { get; protected set; }
+    
+    public virtual Guid AssetCategoryId { get; protected set; }
+    
+    /// <summary>
+    /// Will fall back to <see cref="AssetCategory"/> if the value here is <c>null</c>.
+    /// </summary>
+    public virtual Guid? PeriodSchemeId { get; protected set; }
+    
+    /// <summary>
+    /// This property determines whether assets can be occupied by default when there is no schedule created.
+    /// Will fall back to <see cref="AssetCategory"/> if the value here is <c>null</c>.
+    /// </summary>
+    public virtual AssetSchedulePolicy? DefaultSchedulePolicy { get; protected set; }
+    
```

Comment:
The `Asset` should have `AdvanceBookingDays`  property too.
When it is null, fall back to `AssetCategory` or 'AssetSchedule'.

Reply:
AssetSchedule means "A day of an asset".

So I think the fallback ordering is:
AssetSchedule > Asset > AssetCategory > AssetDefinition